server/controllers/usersController.js

```diff
@@ -26,6 +32,55 @@ class Registered {
     });
   }
 
+//Db controllers
+async create(req, res) {
+  const text = `INSERT INTO users
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)
+    RETURNING *`;
+  const {
+    firstName, lastName, email, password, address, bio, occupation, expertise,
+  } = req.body;
+  const hashPassword = Helper.hashPassword(password);
+  const values = [
+    uuid(),
+    firstName,
+    lastName,
+    email,
+    hashPassword,
+    address,
+    bio,
+    occupation,
+    expertise,
+    moment(new Date()),
+    moment(new Date()),
+  ];
+  try {
+    if (!Helper.isValidEmail(req.body.email)) {
+      return res.status(400).send({ message: 'Please enter a valid email address' });
+    }
+    const newUser = await db.execute(text, values);
```

Comment:
Your controller is doing way too much work. The purpose of controllers should be to pass control to other functions.

Instead, create service files with methods that will communicate with the database and return the response from the database and call that in your controllers

Your service files can be placed in the database folder or create a new folder under `src` and call it services